phy/cluster/manual/session.py

```diff
@@ -689,7 +692,7 @@ def _create_view_model(self, name, **kwargs):
         vm_class = _VIEW_MODELS[name]
         return vm_class(self.model, store=self.cluster_store, **kwargs)
 
-    def show_waveforms(self):
+    def show_waveforms(self, show=True):
```

Comment:
That's too verbose when I, as a user, just want to show the waveforms
ON the other hand, `show_waveforms(show=False)` looks really weird
What about `create_view('waveforms')` so that we just add 1 function? (actually it more or less already exists but it is private, so we make it public)